src/jit/rationalize.cpp

```diff
@@ -324,13 +324,13 @@ Compiler::fgSpliceTreeBefore(BasicBlock* block, GenTreeStmt* insertionPoint, Gen
 }
 
 //------------------------------------------------------------------------------
-// fgMakeEmbeddedStmt: insert the given subtree as an embedded statement 
+// fgMakeEmbeddedStmt: insert the given subtree as an embedded statement
 //
 // Arguments:
 //    block - The block containing the parentStmt, into which the new embedded
 //            statement will go
 //    tree  - The tree that will be the gtStmtExpr of the new embedded statement
-//    parentStmt - The top-level statement that 'tree' is embedded in
+//    parentStmt - A statement (top-level or embedded) that 'tree' is embedded in
```

Comment:
As per my understanding we cannot have an embedded stmnt within another embedded stmnt.  What the above comment be implying is

_Parent statement (top-level or embedded) that the 'tree' is part of_

You might want to check this with @CarolEidt